integration-tests/injecting-multiple-manager-operations-in-a-block.spec.ts

```diff
@@ -4,26 +4,22 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
 
   describe(`Test injecting more than one manager operation in a block: ${rpc}`, () => {
-
     beforeEach(async (done) => {
       await setup();
       done();
     });
 
     it('Verify that doing transfers without awaiting the confirmation after each will fail', async (done) => {
       try {
-        const op1Promise = Tezos.contract.transfer({ to: 'tz1ZfrERcALBwmAqwonRXYVQBDT9BjNjBHJu', amount: 1 });
-        const op2Promise = Tezos.contract.transfer({ to: 'tz1ZfrERcALBwmAqwonRXYVQBDT9BjNjBHJu', amount: 2 });
-
-        const [op1, op2] = await Promise.all([op1Promise, op2Promise])
+        const op1 = await Tezos.contract.transfer({ to: 'tz1ZfrERcALBwmAqwonRXYVQBDT9BjNjBHJu', amount: 1 });
+        const op2 = await Tezos.contract.transfer({ to: 'tz1ZfrERcALBwmAqwonRXYVQBDT9BjNjBHJu', amount: 2 });
 
         await op1.confirmation();
         await op2.confirmation();
-
       } catch (error: any) {
```

Comment:
This code is from earlier, but it does not verify that the code throws. It tests that **IF** it throws, the error is right. But the test can pass if the code just succeeds. I think we need to replace it with:

```
it(‘blahblah’, () => {
  expect(async () => {
    //my code
  }).toThrow(‘error message’);})
```

Reply:
I agree, I'm actually looking to fix this assertion. The current implementation potentially allow this type of practice to fail with different errors. Depending on the timing, it could fail with `key already revealed`, a counter issue, or a mempool conflicting issue. Which sounds unstable, but is actually expected. It hammers the point even more that we'll need to confirm an op before doing another.

Reply:
so it seems that toThrow probably only looks at the first function that's being executed, it doesn't seem like it captures the throw when it's the subsequent methods. I will keep the assertion block as is for now, but we'll investigate sometime in the future.